backend/yugioh/models.py

```diff
@@ -0,0 +1,80 @@
+from django.db import models
+
+from yugiohcardinset.models import YugiohCardInSet
+
+
+class YugiohCard(models.Model):
+
+    type = models.CharField(
+        max_length=100,
+    )
+
+    frame_type = models.CharField(
+        max_length=100,
+    )
+
+    description = models.CharField(
+        max_length=1000,
+        blank=True,
+        null=True,
+    )
+
+    attack = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    defense = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    level = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    race = models.CharField(
+        max_length=100,
+    )
+
+    attribute = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    archetype = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    image = models.URLField(
+        blank=True,
+        null=True,
+    )
+
+    card = models.ForeignKey(
+        'card.Card',
+        on_delete=models.CASCADE,
+    )
+
+    def __str__(self):
+        return self.card
```

Comment:
This returns the `Card` object, which is **not a string**, and therefore throws a `TypeError`. The `__str__` method requires a string to be returned.

- [ ] Make the `__str__` method return the card name (str)

Reply:
fixed